cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -381,6 +381,37 @@ public CloseableIterator<StructuredRow> scan(Range keyRange, int limit, Field<?>
     }
   }
 
+  @Override
+  public CloseableIterator<StructuredRow> scan(Range keyRange, int limit, Field<?> filterIndex, SortOrder sortOrder)
```

Comment:
Please make `public CloseableIterator<StructuredRow> scan(Range keyRange, int limit, Field<?> filterIndex)` delegate here (with ASC order). This two methods are almost same